### PR TITLE
[feat]curvefs/client: split io attr/data latency

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

image

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
cicheck

**Review by SeanHai — APPROVED** [review]

File: curvefs/src/client/fuse_s3_client.cpp [hunk 1]
```diff
@@ -220,6 +217,15 @@ CURVEFS_ERROR FuseS3Client::FuseOpWrite(fuse_req_t req, fuse_ino_t ino,
     }
 
     inodeWrapper->GetInodeAttrLocked(&fileOut->attr);
+
+    if (fsMetric_.get() != nullptr) {
```
Comment (wu-hanqing) [inline curvefs/src/client/fuse_s3_client.cpp:221]: should we remove this metric?

File: curvefs/src/client/fuse_s3_client.cpp [hunk 2]
```diff
@@ -263,17 +271,18 @@ CURVEFS_ERROR FuseS3Client::FuseOpRead(fuse_req_t req, fuse_ino_t ino,
...
         fsMetric_->userRead.qps.count << 1;
         uint64_t duration = butil::cpuwide_time_us() - start;
         fsMetric_->userRead.latency << duration;
         fsMetric_->userReadIoSize.set_value(rRet);
     }
-
-    ::curve::common::UniqueLock lgGuard = inodeWrapper->GetUniqueLock();
-    inodeWrapper->UpdateTimestampLocked(kAccessTime);
-    inodeManager_->ShipToFlush(inodeWrapper);
+    ioLatencyMetric_->readDataLatency << butil::cpuwide_time_us() - mid;
```
Comment (wu-hanqing) [inline curvefs/src/client/fuse_s3_client.cpp:285]: read data latency should put before L274?

File: curvefs/src/client/fuse_s3_client.cpp (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/client/fuse_s3_client.cpp:221]: no, this pr only care about latency.

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**